programs/stake/src/stake_state.rs

```diff
@@ -2351,7 +2351,7 @@ mod tests {
                 &Authorized::default(),
                 &Lockup::default(),
                 &Rent {
-                    lamports_per_byte_year: 42,
+                    lamports_per_byte_year: stake_lamports,
```

Comment:
nit: maybe these can be `Rent::default()` instead?

Reply:
Done!